spaceship/lib/spaceship/tunes/iap_list.rb

```diff
@@ -64,6 +66,22 @@ def edit
       def delete!
         client.delete_iap!(app_id: application.apple_id, purchase_id: self.purchase_id)
       end
+
+      private
+
+      def build_iap
```

Comment:
This method doesn't exist or isn't used, am I missing something?

Reply:
This diff looks strange. I see you pushed some commits from master. I didn't realize you could push commits to my fork. TIL. I have a feeling somehow that's affecting the diff. There were three sets of changes here: 1. Pin jwt dependency in gemspec and then revert that change. 2. Accidentally committed a fixture that gets modified in unit tests and then reverted that. 3. (The only diff that should remain) Add the core_ext for Array with the fl_to_h method. Require that here at the top and then call it below. Not sure why it's showing up this way, but I think that's what you see on the commit before this.

Reply:
Yah, something seems wrong here. I resolved some merge conflicts, maybe it happened because of this?